semanticdb/scalac/library/src/main/scala/scala/meta/internal/semanticdb/scalac/PrinterOps.scala

```diff
@@ -6,25 +6,27 @@ import java.io.StringWriter
 import java.io.Writer
 import scala.collection.mutable
 import scala.reflect.internal.ModifierFlags._
+import `scala`.meta.internal.{semanticdb3 => s}
```

Comment:
This is not the first time I see `scala` backquoted in Scala code. Is this some corner case in the Intellij plugin? In some other tool? Really curious.

Reply:
Probably, somehow I missed this change. I will try to repro and report to Intellij guys

Reply:
It's a ScalaPB bug, `scala` is treated as a keyword https://github.com/scalapb/ScalaPB/blob/f1beff9dee6dfe38494d281a730e544ac520e810/compiler-plugin/src/main/scala/scalapb/compiler/DescriptorPimps.scala#L24

Reply:
I opened a PR updating scalapb https://github.com/scalapb/ScalaPB/pull/447